### PR TITLE
Amend non prod backup window to 30 days from 120

### PR DESCRIPTION
**Pull request by SteveLinden**
Changed 
delete_after       = (local.cold_storage_after + 90)
 to
 delete_after       = local.cold_storage_after

Also tidied up the local name and changed to delete_after. This is only used in here. Removed the send to cold storage as this is no longer needed

### CODE REVIEW

**Review by markgov — APPROVED** [review]
LGTM

File: modules/backup/main.tf [hunk 1]
```diff
@@ -9,10 +9,10 @@ resource "aws_backup_vault" "default" {
 
 # Production backups
 resource "aws_backup_plan" "default" {
-  name = "backup-daily-cold-storage-monthly-retain-120-days"
+  name = "backup-daily-cold-storage-monthly-retain-30-days"
```
Comment (davidkelliott) [inline modules/backup/main.tf:12]:
```suggestion
  name = "backup-daily-retain-30-days"
```

File: modules/backup/main.tf [hunk 2]
```diff
@@ -9,10 +9,10 @@ resource "aws_backup_vault" "default" {
...
 
   rule {
-    rule_name         = "backup-daily-cold-storage-monthly-retain-120-days"
+    rule_name         = "backup-daily-cold-storage-monthly-retain-30-days"
```
Comment (davidkelliott) [inline modules/backup/main.tf:15]:
```suggestion
    rule_name         = "backup-daily-retain-30-days"
```

**Review by ASTRobinson — APPROVED** [review]

**Review by davidkelliott — COMMENTED** [review]
Couple of renaming suggestions so it makes more sensse, could you also update the readme please? - https://github.com/ministryofjustice/modernisation-platform-terraform-baselines/blob/5d802b29b04f6c1d1dbc28f87f2bdb025252c3b6/modules/backup/README.md

**Pull request merged**